scripts/juju-inspect/rules/entities.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2021 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package rules
+
+import "gopkg.in/yaml.v3"
+
+type Report struct {
+	Manifolds map[string]Worker `yaml:"manifolds"`
+}
+
+type Worker struct {
```

Comment:
RFE: differentiate machine and unit agent details.

Reply:
This does this via `--include-nested` :)